packages/api/controllers/operatorServices/operations/saveAlias.js

```diff
@@ -80,11 +80,13 @@ function validateProperty(request, next) {
   const formatPropertyName = /[ `!@#$%^&*()_+\=\[\]{};':"\\|,.<>\/?~]/;
   const formatEnv = /[ `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/;
   const formatUrl = /[ `!@#$%^&*()_+\-=\[\]{};':"\\|,<>\/?~]/;
```

Comment:
Thus seems to be duplicate

Reply:
Actually validations are diff for URL and Env, you can see both are not same. "  .  " is not there in the URL